### ISSUE TITLE
entrypoint issue?

### ISSUE THREAD

**Issue by yann-insa** [thomergil/gopro-map-sync#4]
hi,

maybe  I have a issue , I success to build it's gpxmapmovie from source, but when I when uses I have this log :
C:\Users\yann>docker run -it --mount="type=bind,source=D:/VIDEO/MERCK/OVERLAY/,target=/OVERLAY/" gpxmapmovie  --input /OVERLAY/GS010608.mp4 --output /OVERLAY/GS010608-map.mp4
/usr/bin/env: ‘python\r’: No such file or directory

Orignal dockerfile source, thus : 
# ENTRYPOINT ["/bin/bash"]
ENTRYPOINT ["/usr/bin/pipenv", "run", "/src/gopro-map-sync/gpxmapmovie", "-j", "/src/gpx-animator-1.7/build/libs/gpx-animator-1.7.0-all.jar"]

Please can you help me? I can't understand why it'not functional
Thanks in advance

**Comment by yann-insa** [comment]
more information docker run -it
 : 
`root@0e51f58bf13f:/src/gopro-map-sync# python3 --version
Python 3.9.5
root@0e51f58bf13f:/src/gopro-map-sync# pipenv run ./gpxmapmovie --help
/usr/bin/env: ‘python\r’: No such file or directory
root@0e51f58bf13f:/src/gopro-map-sync#`

**Comment by thomergil** [comment]
I can pay attention to this in a while, but can you try `/usr/bin/env python3` ?

**Comment by yann-insa** [comment]
yes I already test this changement, have you change something with jar argument in gpxmovie file?

**Comment by yann-insa** [comment]
I have find right solution is :

`docker run -it --entrypoint "/usr/bin/env" --mount type=bind,source=D:/VIDEO/MERCK/OVERLAY,target=/OVERLAY/ gpxmapmovie pipenv run python3.9 gpxmapmovie -j /src/gpx-animator-1.7/build/libs/gpx-animator-1.7.0-all.jar   --input /OVERLAY/GS010612.mp4 --output /OVERLAY/GS010612-map.mp4`

Thanks for the help
